config/config.go

```diff
@@ -735,7 +734,7 @@ type MempoolConfig struct {
 // DefaultMempoolConfig returns a default configuration for the Tendermint mempool
 func DefaultMempoolConfig() *MempoolConfig {
 	return &MempoolConfig{
-		Version:   MempoolV0,
+		Version:   MempoolV2,
```

Comment:
[Non Blocking][Question]
Will we be using CAT as soon as its merged by default? shouldn't it be tested for some time first?

Reply:
I have been testing it in testground (and this is the reason why this change exists in the first place) but you're totally right. We should have it opt-in first and v1 be the default

Reply:
I'm ok with being degens and leaving this as the default, wcgw? :cat2:

Reply:
jk jk